### PR TITLE
Edit readme

### PR DESCRIPTION
**Pull request by beccatortell**
I rearranged the readme somewhat for readability. Please confirm that it's still accurate. Thanks.

### CODE REVIEW

**Comment by cep21** [comment]
Can you please install https://github.com/mivok/markdownlint and make sure that ```mdl README.md``` passes?

```
< mdl README.md 
README.md:11: MD009 Trailing spaces
README.md:19: MD009 Trailing spaces
README.md:39: MD009 Trailing spaces
README.md:45: MD009 Trailing spaces
README.md:103: MD009 Trailing spaces
README.md:110: MD009 Trailing spaces
README.md:123: MD009 Trailing spaces
README.md:229: MD009 Trailing spaces
README.md:237: MD009 Trailing spaces
README.md:3: MD013 Line length
README.md:8: MD013 Line length
README.md:11: MD013 Line length
README.md:19: MD013 Line length
README.md:39: MD013 Line length
README.md:45: MD013 Line length
README.md:103: MD013 Line length
README.md:105: MD013 Line length
README.md:123: MD013 Line length
README.md:125: MD013 Line length
README.md:155: MD013 Line length
README.md:207: MD013 Line length
README.md:219: MD013 Line length
README.md:223: MD013 Line length
README.md:229: MD013 Line length
README.md:241: MD013 Line length
README.md:242: MD013 Line length
README.md:252: MD013 Line length
README.md:254: MD013 Line length
README.md:255: MD013 Line length
README.md:223: MD029 Ordered list item prefix
README.md:229: MD029 Ordered list item prefix
README.md:22: MD031 Fenced code blocks should be surrounded by blank lines
README.md:31: MD031 Fenced code blocks should be surrounded by blank lines
README.md:88: MD031 Fenced code blocks should be surrounded by blank lines
README.md:228: MD031 Fenced code blocks should be surrounded by blank lines
README.md:21: MD032 Lists should be surrounded by blank lines
README.md:30: MD032 Lists should be surrounded by blank lines
README.md:229: MD032 Lists should be surrounded by blank lines
README.md:23: MD033 Inline HTML
README.md:: MD033 Inline HTML
README.md:: MD033 Inline HTML
README.md:: MD033 Inline HTML
README.md:6: MD034 Bare URL used
```

**Comment by cep21** [comment]
You should be able to just run ```gem install mdl```

**Comment by mpetazzoni** [comment]
@beccatortell In the code blocks, if they are fenced with the three backquotes, then you don't need leading spaces in them on each line.

**Comment by beccatortell** [comment]
Corrected as many errors as I know how to correct.

File: README.md [hunk 1]
```diff
@@ -16,9 +19,11 @@ signalfx-java will support this.
 
 ## Installation
```
Comment (mpetazzoni) [inline README.md]: "Installation" is actually not super accurate, it's more like "How to use in your project" or something like that.

File: README.md [hunk 2]
```diff
@@ -36,7 +42,16 @@ signalfx-java will support this.
 </dependency>
 ```
 
-### From source (Not recommended)
+To install this library in Scala, add the following to your project's
+`build.sbt` file.
```
Comment (mpetazzoni) [inline README.md]: SBT is not the only build tool for Scala. Maven can be used as well, as described above. I'm sure Gradle has again a different syntax. My point is this section should be structured by build tool, not by language.

**Pull request merged**